### ISSUE TITLE
Devolved content management for resources content type

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#97]
# Done criteria
- [ ] Enable an editor to create/edit a resource
- [ ] Enable an editor to send the resource to an admin for review and to be published
- [ ] Enable the admin to amend and then publish the page

# Designs & Drupal pages
- [Link goes here](http://)
- [Link goes here](http://)

**Comment by meldera** [comment]
Need more info here, i.e. name of the new role and all the differences between the two.

**Comment by angievanhegan** [comment]
@meldera The primary role is admin and the secondary role is editor. Admin has publishing rights, editor is only able to do the stuff specified above. Does that answer your question?

**Comment by meldera** [comment]
Not really, but I've added a "web admin" role (as "administrator" already exist and is exclusive for mrm), and assumed that it should have all the permissions an "editor" has + rights to review/publish resource revisions.

**Comment by angievanhegan** [comment]
@meldera that sounds about right.

**Comment by angievanhegan** [comment]
@meldera Can you give me an editor login please and make sure I'm a web -admin so that I can test this? Thanks very much.

**Comment by angievanhegan** [comment]
@meldera Can you send me details for an editor account please so I can test this (in addition to my angie vanhegan account)

**Comment by meldera** [comment]
@angievanhegan angie@vanhegan.com

**Comment by angievanhegan** [comment]
@meldera You've assigned this to me so I'm assuming it's ready to test? Can you move it to the Manta Ray Review column so I know it's ready? Thanks

**Comment by meldera** [comment]
@angievanhegan this was done a month ago when we spoke. I don't remember the password. Was something really simple. You can reset it: http://bgs-dev.mrmdev.co.uk/user/login

**Comment by angievanhegan** [comment]
@meldera I requested a new password about an hour ago and it's not arrived. I don't know what to do. Can you help please ?

**Comment by meldera** [comment]
qwerty

**Comment by angievanhegan** [comment]
@meldera It doesn't let me log in with username: angie@vanhegan.com and password: qwerty. And it still doesn't let me reset the password myself. It says it has then doesn't send anything. Even to my spam folder.

![screen shot 2018-02-27 at 13 13 47](https://user-images.githubusercontent.com/3796718/36790219-193e2c00-1c62-11e8-8d63-1c7c8cd84f73.png)

**Comment by meldera** [comment]
try username instead (angietheadmin)

**Comment by angievanhegan** [comment]
@StephMSlack I have created this resource with my editor login. Do you want to see if you can publish it as you have a web admin login.

**Comment by StephMSlack** [comment]
@angievanhegan I managed to publish the resource you added. Could I also have an alternate account to test this myself?

**Comment by angievanhegan** [comment]
@dilwarhussain Please can you create an editor account for Steph (in addition to her Web Admin account)? Thanks

**Comment by StephMSlack** [comment]
@angievanhegan @dilwarhussain With this login, I could still edit and move the menu, add all types of content and publish a policy update & a blog. I couldn't publish resource.

How does the BGS web editor get notified that a resource has been created?

**Comment by angievanhegan** [comment]
@StephMSlack I've raised an issue for the first part. Sorry I didn't think to check all of that myself. I'm not sure why you couldn't publish though. 

@dilwarhussain I've added issue 1 above.

**Comment by StephMSlack** [comment]
@angievanhegan @dilwarhussain 
Could we also remove access to the topics and resource type taxonomy from this login type?

I could publish both a news article and policy update (this should be create and save only, no publishing to live site)

Could we remove access to the event content type from this login type?

**Comment by angievanhegan** [comment]
@StephMSlack  Can we look at all the access in this spreadsheet then set them all at the same time? https://docs.google.com/spreadsheets/d/1UUJRooiPsyaohSc-AL9U2ZeWj8H35gS8uGy_fIaCe-4/edit
